infrastructure/helm-chart/charts/tools/charts/akhq/values.yaml

```diff
@@ -1 +1,12 @@
 enabled: false
+workerType:
+hostname:
+namespace: "default"
+kafka:
+  bootstrapServers: "kafka-headless"
+  port: 9092
+  schemaRegistryUrl: "schema-registry"
+  schemaRegistryPort: 8081
+  namespaces:
+    - default
```

Comment:
List of namespaces for which to look for `Kafka` and the `SchemaRegistry`. If the `SchemaRegistry` is not namespaced, then the chart needs to be changed.